classes/task/sendfailednotifications_task.php

```diff
@@ -126,7 +126,9 @@ private function get_failed_email_message($results) {
      *
```

Comment:
missing param comments
what is failedrows, and what does it represent?

Reply:
I have added parameter comments. $failedrows is an array of of failed rows from xapi_failed_log

Reply:
typo: 'array **of of** failed...'

Reply:
The dollar sign is required to denote param names
e.g. `* @param array $failedrows`
https://docs.phpdoc.org/latest/references/phpdoc/tags/param.html

Reply:
I have added the $ sign in the comments.

Reply:
`xapi_failed_log` should be `logstore_xapi_failed_log`

Reply:
I have fixed that.